builtInFunctions/esdtFreezeWipe.go

```diff
@@ -75,7 +79,7 @@ func (e *esdtFreezeWipe) ProcessBuiltinFunction(
 	}
 
 	esdtTokenKey := append(e.keyPrefix, vmInput.Arguments[0]...)
-	identifier, nonce := extractTokenIdentifierAndNonceESDTWipe(vmInput.Arguments[0])
+	identifier, nonce := extractTokenIdentifierAndNonceESDTWipe(e.selfESDTPrefix, vmInput.Arguments[0])
```

Comment:
Is this correct that you always call this func with `e.selfESDTPrefix` ?
If you are on mainnet, `e.selfESDTPrefix` == empty and your func would fail for cross chained assets with prefix

Reply:
- I need to send `e.selfESDTPrefix` so I can validate `vmInput` based on this
- cross-chain assets cannot be wiped, will fail before `ProcessBuiltinFunction` is called